website/content/en/docs/faqs/_index.md

```diff
@@ -346,4 +346,14 @@ where the command `operator-sdk generate kustomize manifests` is not respecting
 made on this manifest. 
 
 [cgo-docs]: https://pkg.go.dev/cmd/cgo
-[open-issue]: https://github.com/operator-framework/operator-sdk/issues/new/choose
\ No newline at end of file
+[open-issue]: https://github.com/operator-framework/operator-sdk/issues/new/choose
+
+## 'operator-sdk run bundle' command fails and the registry pod has an error of 'mkdir: can't create directory '/database': Permission denied'
+
+This error occurs when using `operator-sdk` versions < `v1.22.0` as earlier versions use the deprecated SQLite bundle format by default. 
+By default `operator-sdk run bundle` will use `quay.io/operator-framework/opm:latest` as the index image for creating a registry pod.
+Due to recent pod security updates, using `quay.io/operator-framework/opm:latest` will not work as expected with the SQLite bundle format.
+
+To resolve this, there are two options:
+1. Upgrade to a version of `operator-sdk` >= `v1.22.0` which will use the new FBC bundle format by default
+2. When using `operator-sdk run bundle` specify the index image with `--index-image=quay.io/operator-framework/opm:v1.23.0`
```

Comment:
```suggestion
In version `v1.22.0`, the Operator SDK started using the new file-based catalog (FBC) format by default. Earlier releases used the deprecated SQLite bundle format. The `operator-sdk run bundle` command uses `quay.io/operator-framework/opm:latest` as the index image for creating a registry pod. Due to recent pod security updates, using the latest version of `opm` does not work as expected with the SQLite bundle format.

There are two workarounds available to resolve this issue:
1. You can update the Operator SDK to version `v1.22.0` or later. Updating to a more recent version migrates your project to the new FBC format.
2. If you are not ready to update your version of the Operator SDK or switch to the FBC format, you can manually specify the index image by using the `--index-image=quay.io/operator-framework/opm:v1.23.0` flag.

**Note:** The SQLite bundle format is deprecated and will be removed from the `operator-sdk` command in a future release. If you can, it is recommended that you update the Operator SDK to resolve the issue.
```

Reply:
WDYT?

Reply:
These changes sound good to me. I feel like a couple tweaks to specify that we are only modifying the bundle format when using `operator-sdk run bundle` would be good, but I think that would be pretty easy to work into the wording here. 

Will start updating it now. Thanks @michaelryanpeter !

Reply:
Should be updated in https://github.com/operator-framework/operator-sdk/pull/6054/commits/6fbc6c692756378f08c347fcba649b506901324a